.travis.yml

```diff
@@ -15,7 +15,7 @@ before_install:
   - docker rm -f mysql || true          # in case the after_success step does not run
   - docker pull mysql/mysql-server:5.6
   # port 3306 is not available because Travis runs its own MySQL after 2006-12-01
-  - docker run -p 3307:3306 --name mysql -e MYSQL_ROOT_PASSWORD=leonardo-test -e MYSQL_USER=leonardo-test -e MYSQL_PASSWORD=leonardo-test -e MYSQL_DATABASE=testdb -d mysql/mysql-server:5.6
+  - docker run -p 3307:3306 --name mysql -e MYSQL_ROOT_PASSWORD=leonardo-test -e MYSQL_USER=leonardo-test -e MYSQL_PASSWORD=leonardo-test -e MYSQL_DATABASE=leotestdb -d mysql/mysql-server:5.6
```

Comment:
I changed this because I run a local mysql and this is the same db name as Rawls'.

Reply:
Looks like this is for Travis? Do you want to change `docker/run-mysql.sh` too?

Reply:
oh, good catch